Makefile

```diff
@@ -4,12 +4,17 @@ WHITE=\033[1;37m
 RESET=\033[0m
 
 die:
+	@echo
+	@echo "${WHITE}Everything in red marked with 💥  will destroy the local database.${RESET}"
+	@echo
+	@echo "${WHITE}make install: ${RED}Install and prepare for development. 💥${RESET}"
 	@echo "${WHITE}make reset: ${RED}Destroy and rebuild site. 💥${RESET}"
 	@echo "${WHITE}make site-reset: ${GREEN}Reset site and prepare for development. ${RESET}"
+	@echo
 
 site-reset:
 	@./scripts/site-reset.sh
 
-reset:
+install reset:
```

Comment:
We're not really installing anything so the name is misleading. I can't really decide whether we should have an "install" command at all, or rather a "doctor" like brew that verifies that all dependencies are installed.

Also, site-reset is a bit of a implementation detail, it should be able to assume that the database is in a known state, so I don't think it is a good idea to expose it.

I might be wrong though :)